UT4MasterServer.Web/src/components/Hub.vue

```diff
@@ -3,6 +3,7 @@
         :title="hub?.attributes.UT_SERVERNAME_s" :class="{ 'active-hub': selectedHub?.id === hub?.id }">
         <template v-if="hub">
             <h5>{{ hub.attributes.UT_SERVERNAME_s }}</h5>
+            <div>IP: <span class="hub-ip">{{ hub.serverAddress }}</span></div>

```

Comment:
If you use the <CodeBlock> component you'd get the handy little copy button for free

Reply:
It doesn't look very good IMO

![image](https://user-images.githubusercontent.com/905878/214181277-aa2ba0cf-1c31-4cea-8bd2-08bf1ddf2fe2.png)

Reply:
Eh, you can leave it as is, I'll add the copy button in later

Reply:
Without `IP: `:

![image](https://user-images.githubusercontent.com/905878/214183443-d716c570-64c8-4e60-965d-65aac3e8506f.png)

Reply:
That looks better than than with the IP label, you can go either way, your call

Reply:
I think it looks weird having whole empty line just for COPY on the other side. Not having any extra button and just clicking directly onto the IP could copy it.

Reply:
That's easy enough, @Saibamen you can just leave it as your original and I'll add the click handler to copy if you want

Reply:
IMO adding **auto** click&copy handler will be bad (without any text info ("Click to copy"), because user will not know about updating his clipboard.

Current version is in my first PR comment. Ready for merge